modules/flowable-engine/src/test/java/org/flowable/standalone/initialization/NoDbConnectionTest.java

```diff
@@ -25,21 +27,8 @@
 
     @Test
     public void testNoDbConnection() {
-        try {
-            ProcessEngineConfiguration.createProcessEngineConfigurationFromResource("org/flowable/standalone/initialization/nodbconnection.flowable.cfg.xml").buildProcessEngine();
-            fail("expected exception");
-        } catch (RuntimeException e) {
-            assertTrue(containsSqlException(e));
-        }
-    }
-
-    private boolean containsSqlException(Throwable e) {
-        if (e == null) {
-            return false;
-        }
-        if (e instanceof SQLException) {
-            return true;
-        }
-        return containsSqlException(e.getCause());
+        assertThatThrownBy(() -> ProcessEngineConfiguration
+                .createProcessEngineConfigurationFromResource("org/flowable/standalone/initialization/nodbconnection.flowable.cfg.xml").buildProcessEngine())
+                .hasCauseInstanceOf(SQLException.class);
```

Comment:
This assertion is not the same as before. Before it was checking if the exception itself is `SQLException` or there is a `SQLException` in the stacktrace. I don't know why that was done like that, might be needed for the other DBs perhaps.

Reply:
I think this does the same thing as the manually coded routine; it looks for some exception in the traceback that satisfies a `SQLException` directly or indirectly as an extension.